docs/releases/5.0.txt

```diff
@@ -424,6 +424,11 @@ Miscellaneous
   a page. Having two ``<h1>`` elements was confusing and the site header wasn't
   helpful as it is repeated on all pages.
 
+* On databases without native support for the SQL ``XOR`` operator, the
+  exclusive or (``XOR``) now returns rows that are matched by an odd number of
+  operands rather than exactly one operand. This is consistent with the
+  behavior of MySQL, MariaDB, and Python.
```

Comment:
“The exclusive or” is missing a word: “the exclusive or operator” or “an exclusive or expression” or “an exclusive or query”. Also, we should mention the affected operator by the symbol that appears in user code: `^`.

Reply:
Updated :+1: